regulations/templates/regulations/preamble-partial.html

```diff
@@ -3,10 +3,9 @@
 <section id="{{full_id}}" class="reg-section" data-page-type="preamble-section" data-doc-id="{{doc_number}}">
   <ul id="preamble-read">
     {% render_nested template=sub_template context=sub_context %}
+    {% include "regulations/footnotes.html" %}
```

Comment:
I don't know about *can't*, since this seems to render fine, but yeah. Cool to wrap the include tag in an `<li>`?